explore/src/main/scala/explore/targets/TargetSummaryTable.scala

```diff
@@ -184,7 +198,16 @@ object TargetSummaryTable extends TableHooks:
       // Copy the selection upstream
       .useEffectWithDepsBy((_, _, _, _, table, _, _) =>
         table.getSelectedRowModel().rows.toList.map(_.original.id)
-      )((props, _, _, _, _, _, _) => ids => props.selectedTargetIds.set(ids))
+      )((props, ctx, _, _, _, _, _) =>
+        ids =>
+          props.selectedTargetIds.set(ids) >>
+            ids.headOption
+              .filter(_ => ids.length === 1) // Only if there's just 1 target selected
+              .map(targetId =>
+                ctx.pushPage(AppTab.Targets, props.programId, Focused.target(targetId))
```

Comment:
Does this mean we always see the target summary, even when selecting a target on the left side?

Reply:
> Does this mean we always see the target summary, even when selecting a target on the left side?

No, the target summary is only shown when no asterism or observation is selected.